cmd/podman/create_cli.go

```diff
@@ -121,20 +139,20 @@ func validateVolumeOpts(option string) error {
 	for _, opt := range options {
 		switch opt {
 		case "rw", "ro":
+			foundRWRO++
 			if foundRWRO > 1 {
 				return errors.Errorf("invalid options %q, can only specify 1 'rw' or 'ro' option", option)
```

Comment:
Not from this PR, but do we need this?  Often CLI interfaces allow conflicting options with last-one-wins semantics.

Reply:
We want to throw an error for conflicting options to be compatible with docker.